File: swift/2.swift
var a = 1
var b = 2
var sum = 0

while b < 4_000_000 {
  (a, b) = (b, a + b)

  if a % 2 == 0 {
    sum += a
  }
}

print(sum)
// 4_613_732

File: c/2.c
#include <stdio.h>

int main() {
  int sum = 0;
  int a = 1;
  int b = 1;
  int c = a + b;

  while (c < 4e6) {
    sum += c;
    a = b + c;
    b = c + a;
    c = a + b;
  }

  printf("%d\n", sum);
  return 0;
}
// 4613732
